packages/odoo/test/Adaptor.test.js

```diff
@@ -0,0 +1,89 @@
+import { expect } from 'chai';
+import { enableMockClient } from '@openfn/language-common/util';
+import {
+  execute,
+  create,
+  dataValue,
+  update,
+  read,
+  deleteRecord,
+} from '../src/Adaptor.js';
+
+import MockAgent from './mockAgent.js';
+import { setGlobalDispatcher } from 'undici';
+
+setGlobalDispatcher(MockAgent);
+
+const configuration = {
+  baseUrl: 'https://test.odoo.com/odoo',
+  password: 'somepassword',
+  username: 'someusername',
+  database: 'somedatabase',
+};
+
+const state = { configuration };
+
+describe('read record', () => {
+  it('should read a record', () => {
+    const mock = {
+      read: (model, id, fields) => {
+        expect(model).to.eql('product.product');
+        return {
```

Comment:
If we're passing an array of ids to oodoo, should it not return an array of items back?

Reply:
Yes it is an array